### ISSUE TITLE
no sound

### ISSUE THREAD

**Issue by danielb2** [ankidroid/Anki-Android#4804]
This issue has happened a number of times at this point. only fix seems to be to redownload my decks again. I don't know if the files get corrupted or what, but it works only for awhile after I download everything again. 

###### Research
*Enter an [ x ] character to confirm the points below:*

[ x ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[ x ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum

###### Reproduction Steps

1. try to view any card in a deck

###### Expected Result
sound will play when there's sound

###### Actual Result
no sound plays even when there is sound

###### Debug info
AnkiDroid Version = 2.8.3

Android Version = 8.0.0

ACRA UUID = fdec629e-46ab-4439-ae2b-871d813de64d

**Comment by danielb2** [comment]
I think Google's `File Go` cleaned the files actually.... 

I wonder if this is because the media files are stored outside the working folder?

**Comment by mikehardy** [comment]
I just upgraded my phone and tablet from 7.1 to 8.1 and I am noticing some strange file behavior in other apps and may have seen something similar as this issue as well. I don't have anything concrete to add but I am paying a lot of attention to all the android 8.x issues (there are a few including this one) and anything I can reproduce I'll certainly follow up on and try to patch. Related question: did you see anything like this in Android < 8?

**Comment by mikehardy** [comment]
Doing research, I see this is an app released with 8.1, but is available for previous versions so this could affect the entire userbase.

**Comment by danielb2** [comment]
> did you see anything like this in Android < 8?

yes. And just syncing doesn't make it work again. I had to blow away the whole AnkiDroid folder and start over. And after the sync finished, I had to sync again. Maybe I had to sync twice the first time too, not sure.  But I'm wondering if the syncing itself could be more solid

**Comment by mikehardy** [comment]
Thanks for the extra information. I've installed files go and it looks like it prompts for any deletion so this may be something to note in the faq but I don't think this is actually files go related... There is a separate issue and even a pull request for a sync issue but it doesn't cover disappearing media files...

**Comment by JeroenDeDauw** [comment]
I have also no sound on Android 8.0. Will try to update to 8.1 and if that does nothing, try re-downloading the deck

**Comment by JeroenDeDauw** [comment]
The upgrade to 8.1 did not fix the issue but deleting the deck and re-downloading it worked
